### DISCUSSION TITLE
怎么获取一个点在不在node 的边上

### DISCUSSION THREAD

**Discussion by wenhaoxu1** [antvis/X6#3840] [General]
![image](https://github.com/antvis/X6/assets/107562946/fa617016-c04f-4305-befb-4cdc181fb914)
就像这样,我想鼠标移入节点的边的时候有个提示,并且可以拉一条线出来
![image](https://github.com/antvis/X6/assets/107562946/99d4ca54-0fd5-4e55-9071-648a0f70d9d3)
包括edge可以的soure,和target
![image](https://github.com/antvis/X6/assets/107562946/fcd3c6bc-7120-4082-bb9f-d89c2f966d92)

**Comment by NewByVector** [comment]
可以参考这个例子：https://codesandbox.io/s/lian-jie-dao-ren-yi-wei-zhi-hdd9m7
